sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -434,7 +434,7 @@ as we can guarantee we only need at most two buffers for any call to write (two
              * windowUntil uses a default limit of 256 and once that is hit it will trigger onComplete which causes
              * downstream issues.
              */
-            }, true, parallelTransferOptions.getMaxSingleUploadSize())
+            }, true, Integer.MAX_VALUE)
```

Comment:
Update code comment to align with this change

Reply:
Done

Reply:
Updated